bright-ifit-purchase-validation.yaml

```diff
@@ -181,3 +215,22 @@ paths:
           description: 'Something went wrong when attempting to entitle the user or general internal server error'
       tags:
         - Validate Purchase Transaction
+  /v1/tv-app/user-subscription-info/{userId}:
```

Comment:
not sure that this would be materially different: `GET /v1/tv-app/user/{userId}/subscription-info` is perhaps a lil more rest-y. Maybe not a big deal b/c we prolly aren't anticipating many of these, tho we also prolly weren't anticipating this one either 😅

Reply:
haha sure yeah I was going to ask for some suggestions on path name